assets/scss/_editor.scss

```diff
@@ -185,3 +188,16 @@
     pointer-events: none;
   }
 }
+
+.current-item {
+  position: relative;
+
+  &:after {
+    position: absolute;
+    content: '';
+    border: 2px solid #d9edff;
+    left: -25px;
+    height: 100%;
+    top: 0;
+  }
```

Comment:
What's occurring here?

Reply:
its something I took from the example in the plugin - we may or may not want it, but it seems kind of useful when editing a list/nested list - it highlights the currently scoped list item on which the change will be performed

Reply:
see here https://gitbookio.github.io/slate-edit-list/

Reply:
happy to remove it if you think it's not needed

Reply:
No, I was just curious what it was for. Looks nice to me.